### PR TITLE
java_code_sample

### PR DESCRIPTION
**Pull request by charvakcpatel007**

### CODE REVIEW

**Review by deltasquare4 — COMMENTED** [review]
I fixed the Java example and verified it working on OpenJDK. @charvakcpatel007 Can you please verify it on your end as well? It can be merged once verified working.

**Comment by charvakcpatel007** [comment]
Please commit the gradle folder as well which containers wrapper properties and jar

**Comment by deltasquare4** [comment]
I believe the files you are referring to are already commited by you earlier. I have no changes in those files.

File: java/src/main/java/org/example/SimpleConsumer.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.example;
+
+import org.apache.pulsar.client.api.*;
+
+import java.util.concurrent.ExecutionException;
+
+public class SimpleConsumer {
+    public static String PULSAR_URL = "";
+    public static String PULSAR_JWT = "";
```
Comment (tapanbavaliya) [inline java/src/main/java/org/example/SimpleConsumer.java]: @charvakcpatel007 @deltasquare4 can we update this config same as done for go 
in both `consumer & producer` files
https://github.com/pandio-com/code-samples/blob/master/go/simple-producer-consumer/consumer.go#L13-L14
```
const (
		pulsarURL        string = "{{ pulsar_url }}"
		pulsarJWT        string = "{{ pulsar_jwt }}"
		pulsarTopic      string = "persistent://public/default/topic-go-pandio-go"
		subscriptionName string = "my-subscription-name"
	)
```

**Pull request merged**